python/pyarrow/_parquet.pyx

```diff
@@ -1703,6 +1708,13 @@ cdef shared_ptr[WriterProperties] _create_writer_properties(
     # a size larger than this then it will be latched to this value.
     props.max_row_group_length(_MAX_ROW_GROUP_SIZE)
 
+    # checksum
+
+    if page_checksum_enabled:
+        props.enable_page_checksum()
+    else:
+        props.disable_page_checksum()
```

Comment:
Small naming suggestion: in the Python API, for several other keywords that use a "enable" terminology on the C++ side, we use "write_" or "use_" on the Python side. For example "enable_statistics" on the C++ side is "write_statistics" here. 

So maybe we could also use `write_page_checksum` for the Python user facing keyword.

Reply:
Make sense! I replaced `page_checksum_enabled` with `write_page_checksum` as suggested.